File: testdata/bench/cpp/BenchmarkTest00008.cpp
#include <sstream>
#include <sqlite3.h>

void get_by_name(const std::string &name) {
    std::ostringstream oss;
    oss << "SELECT * FROM users WHERE name = '" << name << "'";
    std::string query = oss.str();
    sqlite3 *db;
    sqlite3_open("app.db", &db);
    sqlite3_exec(db, query.c_str(), nullptr, nullptr, nullptr);
    sqlite3_close(db);
}

File: testdata/bench/swift/BenchmarkTest00008.swift
import Vapor
import SQLite3

func handler008(_ req: Request) throws -> String {
    let apiKey = req.headers.first(name: "X-API-Key") ?? ""
    let query = "SELECT * FROM tokens WHERE key = '\(apiKey)'"
    var db: OpaquePointer?
    sqlite3_open(":memory:", &db)
    sqlite3_exec(db, query, nil, nil, nil)
    return "verified"
}
